From 059a1968743ef612f1db530a4d1a06a13ed2b9e2 Mon Sep 17 00:00:00 2001
From: Jonathan Hamilton <jonathan.hamilton@imgtec.com>
Date: Wed, 22 Jun 2016 11:05:44 -0700
Subject: [PATCH 1/2] Add sgx544 to omap5 device tree

---
 arch/arm/boot/dts/omap5.dtsi | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/arch/arm/boot/dts/omap5.dtsi b/arch/arm/boot/dts/omap5.dtsi
index 6fef4fa..2568c3f 100644
--- a/arch/arm/boot/dts/omap5.dtsi
+++ b/arch/arm/boot/dts/omap5.dtsi
@@ -435,6 +435,17 @@
 			ti,hwmods = "i2c1";
 		};
 
+		sgx:sgx@56000000 {
+			compatible = "ti,dra7-sgx544", "img,sgx544";
+			reg = <0x5600fe00 0x200>;
+			reg-names = "gpu_wrapper";
+			interrupts = <GIC_SPI 16 IRQ_TYPE_LEVEL_HIGH>;
+			ti,hwmods = "gpu";
+			clocks = <&l3_iclk_div>, <&gpu_core_gclk_mux>,
+			         <&gpu_hyd_gclk_mux>;
+			clock-names = "gpu_iclk", "gpu_fclk1", "gpu_fclk2";
+		};
+
 		i2c2: i2c@48072000 {
 			compatible = "ti,omap4-i2c";
 			reg = <0x48072000 0x100>;
-- 
2.9.0

